tools/ds-identify

```diff
@@ -450,7 +449,7 @@ detect_virt() {
 read_virt() {
     cached "$DI_VIRT" && return 0
     detect_virt
-    DI_VIRT=${_RET}
+    DI_VIRT="$(echo "${_RET}" | tr '[:upper:]' '[:lower:]')"
```

Comment:
Just playing safe here, ensuring that DI_VIRT is lower case.

Reply:
If the interpreter is BASH then the following should be possible:

```bash
DI_VIRT="${_RET,,}"
```

Reply:
Nope, can't assume that. sh is bash only in some distros. So, we need to use POSIX complaint method.

Reply:
Oh sure, I was not assuming it. I was saying if we could switch the interpreter explicitly to BASH as you requested elsewhere.

Reply:
Do you happen to know if DASH supports the same pattern?

Reply:
`DI_VIRT="${_RET,,}"` doesn't work on dash. Checked it. It will be nice if we can use bash but it seems like there are distros which doesn't use bash. `#!/bin/sh` is a symbolic link to a posix complaint shell, it can be bash/ksh/dash or anything. Using bash seems not feasible.

Reply:
And yet `local` is used elsewhere, huh? Weird. Seems like if `/bin/sh` is used then the rest of the file should be strictly POSIX compliant...